service-bus.tf

```diff
@@ -0,0 +1,25 @@
+
+module "servicebus-namespace" {
```

Comment:
It looks like from the Terraform module repository we can grab the output `primary_send_and_listen_connection_string` and store in the key vault too - would be worth doing this if it works! We can store it as a key named `hmc-servicebus-connection-string`.